scripts/items/moogle_cap.lua

```diff
@@ -5,20 +5,23 @@
 -----------------------------------
 local itemObject = {}
 
+local pos =
+{
+    -- Content  [Nation] = {   x,   y,  z, rot, zone },
+    [xi.nation.SANDORIA] = { 126,   0,  -1, 122, 231 },
+    [xi.nation.BASTOK  ] = { 106,   0, -71, 130, 234 },
+    [xi.nation.WINDURST] = { 197, -12, 224,  65, 240 },
+}
+
 itemObject.onItemCheck = function(target)
     return 0
 end
 
 itemObject.onItemUse = function(target)
-    local nation = target:getNation(target)
-
-    if nation == 0 then -- San d'Oria
-        target:setPos(126, 0, -1, 122, 231)
-    elseif nation == 1 then -- Bastok
-        target:setPos(106, 0, -71, 130, 234)
-    elseif nation == 2 then -- Windurst
-        target:setPos(197, -12, 224, 65, 240)
-    end
+    target:timer(1000, function(playerArg)
```

Comment:
I don't think timer() is the best way to do this. There's a delay baked into a status effect we're supposed to use, and the set pos need to be moved to it. For all items like this. They need their destinations added to the teleports global and enumerated.

Reply:
Agreed, was just trying to take the path of least resistance. Suppose we can make it a proper teleport using xi.effect.teleport but that would need to happen after my campaign PR gets merged.

Also it is literally 3 items that have this paradigm

Reply:
also 2 of the 3 were able to use the pre-existing teleport type 👍